tests/test_admin.py

```diff
@@ -482,3 +482,49 @@ def test_versions_filters_all_state(self):
             transform=lambda x: x.pk,
             ordered=False,
         )
+
+
+class ContentExpiryCsvExportTestCase(CMSTestCase):
+    def test_export_button_endpoint_response_is_a_csv(self):
+        """
+        Valid csv file is returned from the admin export endpoint
+        """
+        date = datetime.datetime.now() - datetime.timedelta(days=5)
+        PollContentExpiryFactory(expires=date, version__state=DRAFT)
+
+        version_selection = "?state=_all_"
+        export_endpoint = "export_csv/"
+        admin_endpoint = self.get_admin_url(ContentExpiry, "changelist")
+
+        with self.login_user_context(self.get_superuser()):
+            response = self.client.get(admin_endpoint + export_endpoint + version_selection)
+
+        # Endpoint is returning 200 status code
+        self.assertEqual(response.status_code, 200)
+        # Response contains a csv file
+        self.assertEquals(
+            response.get('Content-Disposition'),
+            "attachment; filename={}.csv".format("djangocms_content_expiry.contentexpiry")
+        )
```

Comment:
We should look into the file here and check that the Poll is actually in there and the headings, the headings would need to be a separate test.